layouts/_default/baseof.html

```diff
@@ -62,22 +62,20 @@
                         <li class="dropdown-header">References</li>
                         <li><a class="dropdown-item"  href="/documentation/index.html">Overview</a></li>
                         <li><a class="dropdown-item"  href="/documentation/query/index.html">ARQ (SPARQL)</a></li>
+                        <li><a class="dropdown-item"  href="/documentation/io/">RDF I/O</a></li>
                         <li><a class="dropdown-item"  href="/documentation/assembler/index.html">Assembler</a></li>
                         <li><a class="dropdown-item"  href="/documentation/tools/index.html">Command-line tools</a></li>
                         <li><a class="dropdown-item"  href="/documentation/rdfs/">Data with RDFS Inferencing</a></li>
                         <li><a class="dropdown-item"  href="/documentation/geosparql/index.html">GeoSPARQL</a></li>
                         <li><a class="dropdown-item"  href="/documentation/inference/index.html">Inference API</a></li>
-                        <li><a class="dropdown-item"  href="/documentation/javadoc.html">Javadoc</a></li>
```

Comment:
Should this link also be removed, I think this is the general java docs landing page, and I don't see it moved anywhere else?